cmd/codeqlExecuteScan.go

```diff
@@ -220,6 +239,15 @@ func runCodeqlExecuteScan(config *codeqlExecuteScanOptions, telemetryData *telem
 
 	cmd = nil
 	cmd = append(cmd, "database", "analyze", "--format=csv", fmt.Sprintf("--output=%vtarget/codeqlReport.csv", config.ModulePath), config.Database)
+	if len(config.Threads) > 0 {
+		cmd = append(cmd, "--threads="+config.Threads)
+	} else {
+		cmd = append(cmd, "--threads=0")
```

Comment:
We have the default set. I think we can remove this 'else' everywhere now.

Reply:
You're right, thanks. Fixed it